cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -4550,6 +4550,50 @@ action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = VPN
 
 
+
+[Authentication - Failed VPN Login From Unusual Country]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 59 * * * *
+description = A Failed VPN Login From Unusual Country. \
+\
+Data Collection - VPN data mapped with authentication data-model and has dest_category=vpn_auth.
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = | tstats `cs_summariesonly_authentication` count from datamodel=Cyences_Authentication where Authentication.dest_category="vpn_auth" AND Authentication.action="failure" AND `cs_public_ips(Authentication.src)` by Authentication.app, Authentication.user, Authentication.src, Authentication.dest, _time \
+| `drop_dm_object_name(Authentication)` \
+| eval user = lower(user) \
+| iplocation src \
+| `cs_current_week_login_count(pan:globalprotect)` \
+| `cs_country_login_percentage` \
+| `cs_user_privilege_mapping(user)` \
+| eval cyences_severity = case(percentage_login_from_country!=0 and is_privileged_user=="Yes", "critical", true(), "high") \
```

Comment:
Replace the **percentage_login_from_country!=0** ==> **percentage_login_from_country==0** which means till now no logins from the country.